src/command.h

```diff
@@ -183,6 +187,52 @@ class DrawRectCommand : public PipelineCommand {
   float height_ = 0.0;
 };
 
+/// Command to draw a grid of recrangles on screen.
+class DrawGridCommand : public PipelineCommand {
+ public:
+  explicit DrawGridCommand(Pipeline* pipeline, PipelineData data);
+  ~DrawGridCommand() override;
+
+  const PipelineData* GetPipelineData() const { return &data_; }
```

Comment:
This is always empty, can we remove it?

Reply:
Copied off draw_rect. Removed here, left in draw_rect.